packages/botonic-react/src/webchat/header.jsx

```diff
@@ -40,18 +52,25 @@ export const DefaultHeader = props => (
     </Flex>
     <Flex width={1} flexDirection='column'>
       <HeaderTitle>Botonic</HeaderTitle>
-      <Subtitle>Online</Subtitle>
     </Flex>
+    <CrossHeader onClick={props.onChange}>X</CrossHeader>}
```

Comment:
If the component is called "CrossHeader" I would expect the X character to be included in the component